CONTRIBUTING.md

```diff
@@ -44,13 +44,12 @@ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 # Configuring apt to find nodejs
 curl -sL https://deb.nodesource.com/setup_18.x | sudo -E bash -
 
-# Configuring apt to find yarn
-curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | gpg --dearmor | sudo tee /usr/share/keyrings/yarnkey.gpg > /dev/null
-echo "deb [signed-by=/usr/share/keyrings/yarnkey.gpg] https://dl.yarnpkg.com/debian stable main" | sudo tee /etc/apt/sources.list.d/yarn.list > /dev/null
+# Configuring apt to find pnpm
+curl -fsSL https://get.pnpm.io/install.sh | sh -
```

Comment:
You configure apt here to find pnpm and then dont use the apt pnpm install?
Perhaps rather just install pnpm via npm?

```
npm -g install pnpm
```

Reply:
The only disadvantage would be that if you use `nvm` or something else to handle node versions, you would be required to reinstall pnpm.

Reply:
hmm, i suppose. Not really a huge concern since we only use node 18 but regardless seems fine to do it via apt. Just add the apt install for pnpm part then its fine.

Reply:
@EsdrasAmora I do use nvm, not the end of the world if i have to reinstall pnpm